tox.ini

```diff
@@ -2,7 +2,7 @@
 # Tox global configuration
 #
 [tox]
-envlist = py37,py{38,39,310}-nocover
+envlist = py37-nocover,py{38,39,310}
```

Comment:
Based on the Debian 10 CI run of this branch and https://github.com/apache/buildstream/pull/1420#issuecomment-773094285, coverage should also work with Python 3.7. Have you encountered issues with coverage in Python 3.7?

Reply:
I don't remember, I'll try to change `common.env` and see